sdk/python/kfp/compiler/_default_transformers.py

```diff
@@ -30,12 +50,23 @@ def add_pod_env(op: BaseOp) -> BaseOp:
             )
         ).add_env_variable(
             k8s_client.V1EnvVar(
-                name='KFP_NAMESPACE', 
+                name='KFP_NAMESPACE',
                 value_from=k8s_client.V1EnvVarSource(
                     field_ref=k8s_client.V1ObjectFieldSelector(
                         field_path='metadata.namespace'
                     )
                 )
             )
         )
-    return op
\ No newline at end of file
+    return op
+
+
+def add_pod_labels(labels: Optional[Dict[Text, Text]] = None) -> BaseOp:
+    """Adds random pipeline uuid to each pod."""
+
+    def _add_pod_labels(task):
```

Comment:
Should this be Kubernetes label or just annotation?

Reply:
Synced offline. Let's use k8s label for now.